### PR TITLE
[Doc improvement][Core capabilities intents to actions][3334699]

### PR DESCRIPTION
**Pull request by v-ypalikila**
Updated Intents to Actions.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c71e4c4174b22eb44a01dadb9c1a8ce947701a96/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da69222a-910b-4470-a8c9-37059252379b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b36742d3b93d2de98cd047f81b7befa530f25f1c/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/27e2ce84-c1a9-49ab-8a06-089361f2187e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed87f510cb14351fab19946b4535714af7f1da7c/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/59625ae1-acb5-4e3c-829c-28ffff8ad595) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a8edb26ec9b6dd9051bac95813f1c61a68209b9/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/046a248a-5dff-4445-a46b-0999a7ae7bce) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9a5d34a1367afa03c1435e5fd8fa23bdc05b27b4/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/45b584e1-337d-41d3-a3ca-bb6d7fd299fb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/15946baac06ad2c51a0c1a042942074d6395a680/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f8617fb1-ef05-42ee-86b7-9c7568d3454d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/15946baac06ad2c51a0c1a042942074d6395a680/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 93 | 39 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1815edc9-fc82-4419-a24c-8bb2d252e933) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by singhk97 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2574fd9d32b56f4afed5cf7a93778790767495a8/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ecf7f40-f64f-4b10-bf7f-905b4a5b5a0f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2574fd9d32b56f4afed5cf7a93778790767495a8/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f303d6ef-73b5-4daa-9123-781a538f50a5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a774b5584983caa549f7279e7fb41e62f00e29e2/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aabc596a-1f19-4091-a90e-7df70b85c44f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a774b5584983caa549f7279e7fb41e62f00e29e2/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 93 | 39 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46b977d2-db1b-4e1e-88b3-c1ee8f38b842) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ee83132d861997681e7f87389b844ae21d0f8da4/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e590cb43-cace-46d3-8704-9f827fe136af) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ee83132d861997681e7f87389b844ae21d0f8da4/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 93 | 39 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/44b8cdbe-0c3f-4758-ac07-40d05502c132) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by govil1811 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fc1787561ee39632e8df83c382c62a3402ea7656/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/069dc099-2428-48ba-b69a-80b79859abe7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fc1787561ee39632e8df83c382c62a3402ea7656/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 93 | 39 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c07dde8e-9aaa-4712-9a58-d3b4e7db65cd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b1c3e8574bb53879f40968be3e3cc5791925017e/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/484fd0bb-96a8-47bb-ae25-246d03b979e7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b1c3e8574bb53879f40968be3e3cc5791925017e/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 93 | 39 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/882887f7-a419-4857-b9bb-90ec15ce7d10) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md [hunk 1]
```diff
@@ -285,73 +285,112 @@ A simple interface for actions and predictions allows bots to react when they ha
 
 Thanks to our AI library, the prompt needs only to outline the actions supported by the bot, and supply a few-shot examples of how to employ those actions. Conversation history helps with a natural dialogue between the user and bot, such as *add cereal to groceries list*, followed by *also add coffee*, which should indicate that coffee is to be added to the groceries list.
 
-All entities are required parameters to actions.
+The following is a conversation with an AI assistant. The assistant can manage lists and supports the following commands:
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md:288]: The AI assistant is capable of managing lists and recognizes the subsequent commands:

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md [hunk 2]
```diff
@@ -285,73 +285,112 @@ A simple interface for actions and predictions allows bots to react when they ha
...
-    if (index >= 0) {
-        await context.sendActivity(responses.itemFound(data.list, data.item));
-    } else {
-        await context.sendActivity(responses.itemNotFound(data.list, data.item));
-    }
+    ```javascript
+    {{conversation.listNames}}
+    ```
 
+* AI: The bot logic is simplified to provide handlers for actions such as `addItem`, `removeItem`, and `findItem`. This clear delineation between actions and the prompts that instruct the AI on how to execute them is an incredibly potent tool.
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md:349]: The bot logic is streamlined to include handlers for actions such as `addItem`, `removeItem`, and `findItem`. This distinct separation between actions and the prompts guiding the AI on their execution serves as a powerful tool.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ import path from "path";
 
 Add AI capabilities to your existing app or a new Bot Framework app.
 
-**Planner**: OpenAI planner is the main component that calls the large language model (LLM) OpenAI or Azure OpenAI. The OpenAI API is powered by a diverse set of models with different capabilities. You can also make limited customizations to our original base models for your specific use case.
+**Planner**: A planner is a function that takes in a user's ask and returns a plan on how to accomplish the request. Planner is the main component that calls the large language model (LLM) OpenAI or Azure OpenAI. The OpenAI API is powered by a diverse set of models with different capabilities. You can also make limited customizations to our original base models for your specific use case.
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md:71]: **Planner**: A planner is a feature that accepts a user's request and generates a strategy to fulfill it. The planner is the primary component that invokes the Large Language Model (LLM) from OpenAI or Azure OpenAI. The OpenAI API is driven by an array of models, each with unique capabilities. For your specific use case, you might make minor customizations to our original base models.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md [hunk 4]
```diff
@@ -204,33 +204,15 @@ The language supports features that allow you to include variables, call externa
...
-    await context.sendActivity(`I'm sorry your message was flagged: ${JSON.stringify(data)}`);
-    return false;
-});
-
-app.ai.action(AI.FlaggedOutputActionName, async (context: TurnContext, state: ApplicationTurnState, data: TData) => {
-    await context.sendActivity(`I'm not allowed to talk about such things.`);
-    return false;
-});
-```
+Actions are atomic functions that are registered to the AI Module and serve as the fundamental building blocks of a plan. Actions allow you to define the specific tasks your app will perform and are pivotal in executing plans and enhancing your application's functionality. For instance, you might want your app to create a list, send a message, or perform a calculation. By creating and registering actions, you can control these behaviors and enhance your app's functionality.
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md:207]: Actions serve as fundamental building blocks of a plan, functioning as atomic tasks registered to the AI Module. They define the specific tasks your app performs, playing a crucial role in executing plans and boosting your app's functionality. For example, you might want your app to generate a list, dispatch a message, or carry out a calculation. By crafting and registering actions, you gain control over these behaviors, thereby improving your app's functionality.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md [hunk 5]
```diff
@@ -204,33 +204,15 @@ The language supports features that allow you to include variables, call externa
...
 
 ### Register Action Handlers
 
-Action handlers help users achieve the goals, which is shared in the user intents.
-
-One of the key aspects in action handlers is that you must first register the actions in the prompts and then help user achieve the goal.
+Action handlers help users achieve the goals, which is shared in the user intents. One of the key aspects in action handlers is that you must first register the actions in the prompts and then help user achieve the goal.
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md:211]: Action handlers assist users in accomplishing their intended goals. A crucial aspect of action handlers is the initial registration of actions in the prompts, followed by aiding the user in goal achievement.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md [hunk 6]
```diff
@@ -204,33 +204,15 @@ The language supports features that allow you to include variables, call externa
...
 
-You must register a handler for each action listed in the prompt and also add a handler to deal with unknown actions.
+You must register a handler for each action listed in the prompt and also add a handler to deal with unknown actions. You can use the `app.ai.action` method to register an action.
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md:213]: Ensure you register a handler for every action specified in the prompt. Additionally, incorporate a handler to manage unknown actions. Utilize the `app.ai.action` method for action registration.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md [hunk 7]
```diff
@@ -204,33 +204,15 @@ The language supports features that allow you to include variables, call externa
...
 
-In the following example of a light bot, we have the `LightsOn`, `LightsOff`, and `Pause`  action. Every time an action is called, you return `true` or `false`. ​Returning `false` from a handler prevents the planner from running additional `DO` or `SAY` commands. When the bot receives an unknown action, we're telling the bot to terminate the action.
+In the following example of a light bot, we have the `LightsOn`, `LightsOff`, and `Pause`  actions. Every time an action is called, you return `true` or `false`. ​Returning `false` from a handler prevents the planner from running additional `DO` or `SAY` commands. When the bot receives an unknown action, we're telling the bot to terminate the action.
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md:215]: In the given light bot example, the actions `LightsOn`, `LightsOff`, and `Pause` are present. Each time an action is invoked, it returns either `true` or `false`. If a handler returns `false`, it stops the planner from executing further `DO` or `SAY` commands. If the bot encounters an unknown action, it is instructed to terminate that action.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md [hunk 8]
```diff
@@ -265,6 +247,24 @@ app.ai.action(
 );
 ```
 
+### Moderation
+
+`FlaggedInputAction` and `FlaggedOutputAction` are the built-in action handlers to handle the moderator flags. If the moderator flags an incoming message input, the moderator redirects to the `FlaggedInputAction` handler and the `context.sendActivity` sends a message to the user about the flag.
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md:252]: `FlaggedInputAction` and `FlaggedOutputAction` serve as the integrated action handlers for managing moderator flags. When a moderator flags an incoming message input, the system redirects the moderator to the `FlaggedInputAction` handler. The `context.sendActivity` then dispatches a message to the user, notifying them about the flag.

**Review by v-deabira — CHANGES_REQUESTED** [review]
@v-ypalikila, lets connect on the suggestions.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2e6148e7185d4d4f03dee4780ea069c5913c416a/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89afaac4-9139-4c8f-954c-e044339f23c6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2e6148e7185d4d4f03dee4780ea069c5913c416a/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 92 | 40 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/af653543-33ca-41b3-a38f-7ab3b3d49f09) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md (hunk 8 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md]: Updated.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md (hunk 7 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md]: updated.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md (hunk 6 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md]: Updated.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md (hunk 5 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md]: Updated.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md (hunk 3 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md]: Updated.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md (hunk 2 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md]: Updated.

File: msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md]: Updated.

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/80d159d5201d374a7530db7b51e334cfffc908c6/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be83c404-3f23-43bc-a2f4-a8885df1e4f3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/80d159d5201d374a7530db7b51e334cfffc908c6/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 92 | 40 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd892588-ed1d-4d6c-a893-d649b697c263) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**